pyodide_build/pywasmcross.py

```diff
@@ -221,14 +223,18 @@ def parse_args():
     parser.add_argument(
         '--target', type=str, nargs='?', default=common.TARGETPYTHON,
         help='The path to the target Python installation')
-    args = parser.parse_args()
-    return args
+    return parser
 
 
-if __name__ == '__main__':
+def main(args, unknown=None):
     basename = Path(sys.argv[0]).name
     if basename in symlinks:
         collect_args(basename)
     else:
-        args = parse_args()
         build_wrap(args)
+
+
+if __name__ == '__main__':
+    parser = make_parser(argparse.ArgumentParser())
```

Comment:
Yes, it's a but tricky, because due to the fact that we gave the main parser, to make it work an `ArgumentParser` object need to be provided in any case. I tried to improve the situation a bit and skip all parsing in this case.

For instance, `tools/gcc -h` will still yield the wrong help, it should be disabled but it appears not to be, but everything else we are directly sending arguments to gcc etc.